Examples/Examples.m

```diff
@@ -36,6 +36,8 @@ + (NSArray *)list {
         MBXExampleFillPattern,
         MBXExampleImageSource,
         MBXExampleLight,
+        MBXExampleFillPattern,
```

Comment:
This already exists three lines up, so you can safely delete this one.

Reply:
Oops 😅